### PR TITLE
Allow PeerInfo names

### PR DESCRIPTION
**Pull request by juslesan**
Supporting PR for https://github.com/streamr-dev/broker/pull/164

### CODE REVIEW

File: src/connection/PeerInfo.js [hunk 1]
```diff
@@ -5,19 +5,19 @@ const peerTypes = Object.freeze({
...
     }
 
-    static newStorage(peerId) {
-        return new PeerInfo(peerId, peerTypes.STORAGE)
+    static newStorage(peerId, peerName) {
+        return new PeerInfo(peerId, peerTypes.STORAGE, peerName)
     }
 
-    constructor(peerId, peerType) {
+    constructor(peerId, peerType, peerName = 'StreamrNetworkNode') {
```
Comment (harbu) [inline src/connection/PeerInfo.js]: Let's just use empty string as default name? No name given => use ID as "name".

**Review by harbu — COMMENTED** [review]
What about exchanging name info with other nodes on connection?

**Comment by juslesan** [comment]
> What about exchanging name info with other nodes on connection?

I will add a separate PR for this

**Review by mirpo — APPROVED** [review]

**Review by harbu — APPROVED** [review]
LGTM

**Pull request merged**